### ISSUE TITLE
Please merge "JarFileResource improve performance of exist method" to 9.2 version

### ISSUE THREAD

**Issue by andreysubbotin** [eclipse/jetty.project#1678]
See https://github.com/cuba-platform/jetty.project/commit/7da5db8dc9e1b6a0a4683085cac7e5217ef66d24

**Comment by gregw** [comment]
@jmcc0nn3ll what is the current state of 9.2?  Are we doing open source releases still?

**Comment by joakime** [comment]
9.2.x has not even been announced for EOL yet.
When Java 9 is officially released is when the EOL announcement will occur (set for 6 months after announcement to be officially EOL).

In summary: Jetty 9.2 EOL is Java 9 official release date + 6 months.

**Comment by gregw** [comment]
@joakime I'm not sure why you think 9.2 EOL is related to java 9 release?    Jetty 9.2 is Java 7 and that has been EOL since April 2015.   9.4 is current stable version for java 8 and it will live on as long as Java 8 does.

@jmcc0nn3ll can you confirm where we are at with 9.2

**Comment by jmcc0nn3ll** [comment]
Last 9.2.x release was in June and it is still lingering around as client supported, we'll patch for security issues of course but doing little to no open source support on it unless it is something critical.  The thinking with regards to java9 + 6mo is that while java 7 has been EOL we still see a fair amount of people using it.  Once java 9 has landed the point of java7 being 2 full versions behind makes it much harder to rationalize keeping any support around for so might as well make it official plus give the 6 month notice.  

TL;DR We need an official announcement that 9.2.x is fully Community EOL and the release of java 9 seemed a good time to mark that occasion.

@andreysubbotin As a user, can you explain your rationale for still using 9.2.x and java 7?  If you are using java 8, can you explain why you would still be on 9.2.x?  It is 2 major versions behind at this point and your reasoning here would be helpful, sorry to draft you in like this. :)

**Comment by gregw** [comment]
This is not going to happen now 9.2 is EOL.
